kyo-prelude/shared/src/main/scala/kyo/Stream.scala

```diff
@@ -328,25 +329,52 @@ final case class Stream[V, -S](v: Ack < (Emit[Chunk[V]] & S)):
 end Stream
 
 object Stream:
+    @nowarn("msg=anonymous")
+    inline def apply[V, S](v: => Ack < (Emit[Chunk[V]] & S)): Stream[V, S] =
```

Comment:
we can avoid the function allocation for `v`:

```suggestion
    inline def apply[V, S](inline v: => Ack < (Emit[Chunk[V]] & S)): Stream[V, S] =
```